### PR TITLE
Fixing kbd layout issues and non-logical crashes in guided.py questions

### PR DESCRIPTION
**Pull request by advaithm**
🚨 PR Guidelines:

# Describe your PR

This pr fixes the legitimate issues, raised in https://youtu.be/Yc5Fs8sfJHg. we should have better error handling in guided.py
# Testing

Any new feature or stability improvement should be tested if possible.
Please follow the test instructions at the bottom of the README.

*These PR guidelines will change after 2021-05-01, which is when `v2.1.4` gets onto the new ISO*

### CODE REVIEW

File: archinstall/lib/user_interaction.py [hunk 1]
```diff
@@ -347,16 +347,17 @@ def select_language(options, show_only_country_codes=True):
...
 		# all possible language layouts, and we might want to write
 		# for instance sv-latin1 (if we know that exists) without having to
 		# go through the search step.
 		elif selected_language in options:
 			selected_language = options[options.index(selected_language)]
+			return selected_language
 		else:
-			RequirementError("Selected language does not exist.")
-		return selected_language
+			print("Invalid Langue please select a valid option.")
```
Comment (Torxed) [inline archinstall/lib/user_interaction.py:360]: Language?

File: examples/guided.py [hunk 2]
```diff
@@ -287,11 +291,16 @@ def perform_installation(mountpoint):
 
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
+<<<<<<< HEAD
+			if archinstall.arguments['mirror-region'] != None:
+				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
+=======
```
Comment (Torxed) [inline examples/guided.py:297]: I think this is a automerge issue hehe

File: archinstall/lib/user_interaction.py (hunk 1 above)
Comment (advaithm) [reply archinstall/lib/user_interaction.py:360]: my bad, vscode lacks a spell check

File: examples/guided.py [hunk 3]
```diff
@@ -291,16 +291,8 @@ def perform_installation(mountpoint):
...
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
-<<<<<<< HEAD
 			if archinstall.arguments['mirror-region'] != None:
 				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
-=======
-
-			# Configure the selected mirrors in the installation
-			if archinstall.arguments.get('mirror-region', None):
-				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
```
Comment (Torxed) [inline examples/guided.py]: This is from `master` and should stay : )

Comment (advaithm) [reply examples/guided.py]: it can't really stay because of how the we return mirrors now

File: examples/guided.py (hunk 2 above)
Comment (advaithm) [reply examples/guided.py]: I've fixed, yeah gits automerge can be a bit funky

**Comment by Torxed** [comment]
[this change](https://github.com/archlinux/archinstall/blob/12b43f443bc128bb1eb3f53484eaf4449bd7d4d4/archinstall/lib/locale_helpers.py) should fix #149 issues with non-qwerty locales. This wasn't documented in the issue mentioned but in a youtube video describing the issue this was shown.

**Pull request merged**